packages/web-vue/components/select/select.tsx

```diff
@@ -533,6 +533,23 @@ export default defineComponent({
       ...fieldNames?.value,
     }));
 
+    // selected label
+    const _selectedLabel = ref();
```

Comment:
Here can directly cache the data of the original option, and then add the cached data in `extraOptions`, only add a label alone will cause the option data exported by some custom functions to not match. In this way, the cached data can be shared with other function. For example, the `formatLabel` function will export the option data as a param, so that the cached data will be exported here.

Reply:
@flsion Thanks for your suggestion. I just updated this PR. PTAL again. Thanks. 🙇

Reply:
@flsion Thanks for your suggestion. I just updated this PR. PTAL again. Thx. 🙇